packages/breadcrumb-react/documentation/Breadcrumb.mdx

```diff
@@ -0,0 +1,89 @@
+
+<Ingress>
+    En brødsmulesti brukes der man har en trestruktur i navigasjonen, og det er behov for å synliggjøre hvor i denne
+    strukturen brukeren befinner seg.
+</Ingress>
+
+<ComponentExample
+    component={BreadcrumbExample}
+    knobs={{ boolProps: ["Compact"] }}
+    codeExample={breadcrumbExampleCode}
+/>
+
+Brødsmulestien vises typisk høyt oppe på siden, gjerne rett under headeren. Dersom en brødsmulesti brukes i applikasjonen skal den vises på alle sider som ikke er forsiden. Det siste elementet i brødsmulestien skal **ikke** være klikkbart. De andre elementene skal være lenker.
+
+## Tilgjengelighet
+
+En brødsmulesti har potensielt mange interaktive elementer som en tastaturbruker lett må komme seg forbi. Det er derfor viktig at [skip-link](/universell-utforming/guide#uu/hoppe-over-blokker) til hovedinnholdet også hopper over brødsmulestien.
+
+## Når bruker vi Breadcrumb?
+
+Vi bruker en brødsmulesti når det er en trestruktur i navigasjonen – tenk mapper.
+
+Det blir typisk et behov først når det er tre eller flere nivåer. Likevel, dersom en brødsmulesti brukes i applikasjonen overhode skal den vises på alle sider som ikke er forsiden – altså selv på nivå to.
+
+Vi bruker **ikke** brødsmulestier når det er en lineær navigasjon, for eksempel for de ulike stegene i en kjøpsflyt eller et oppsigelsesskjema.
+
+Om det bare er to nivåer i strukturen bruker vi typisk en _back_-variant av [NavLink](/komponenter/link).
+
+## Eksempel med router
+
+Jøkul-komponentene er uavhengige av _router_. Et tips er å lage en komponent i ditt eget prosjekt som bruker Jøkul-komponenter sammen med eventuelle `Link`-komponenter og _smartness_ som tilbys av _router_-biblioteket du bruker.
+
+**Del ditt eksempel!** Om du har et eksempel for en annen _router_, eller for annen major-versjon, så er det en stor hjelp for kollegene dine om du deler. Åpne en pull request og vis fram hvordan du har gjort det.
+
+### React Router DOM
+
+**Versjon 6**
+
+```tsx
+import React, { useMemo } from "react";
+import type { RouteObject } from "react-router-dom";
+import { Link, matchRoutes, useMatch, useLocation } from "react-router-dom";
+import { Breadcrumb, BreadcrumbItem } from "@fremtind/jkl-breadcrumb-react";
+
+export interface MyBreadcrumbProps {
+    routes: RouteObject[];
+}
+
+export const MyBreadcrumb = ({ routes }: MyBreadcrumbProps) => {
+    const location = useLocation();
+
+    const matches = useMemo(() => {
+        const routeMatches = matchRoutes(routes, location);
+        return [{ pathname: "/", route: { element: "Hjem" } }, ...routeMatches];
+    }, [location]);
+
+    const isHjem = useMatch("/");
+
+    if (isHjem) {
+        return null;
+    }
+
+    return (
+        <Breadcrumb>
+            {matches.map((match, i) => {
+                return (
+                    <BreadcrumbItem key={match.pathname} isLastElement={i === matches.length - 1}>
+                        {isLastElement ? (
+                            match.route.element
```

Comment:
Ettersom [eksempelet til W3 for en tilgjengelig brødsmulesti](https://www.w3.org/TR/wai-aria-practices/examples/breadcrumb/index.html) også har siste-elementet som en link (selv om den ikke skiller seg ut som det) valgte jeg å gjøre det sånn i vår løsning. Har du gjort deg noen tanker om dette?

Reply:
Hmm, har du sett. Ser Carbon gjør det samme. Det la jeg faktisk ikke merke til – antok det var tekst siden det var "ustylet".